pkg/library/ssh/event.go

```diff
@@ -22,11 +22,11 @@ import (
 )
 
 const commandLine = `SSH_ENV_PATH=$HOME/ssh-environment \
-&& if [ -f /etc/ssh/passphrase ] && command -v ssh-add >/dev/null; \
+&& if [ -f /etc/ssh/passphrase ] && [ -w $HOME ] && command -v ssh-add >/dev/null; \
 then ssh-agent | sed 's/^echo/#echo/' > $SSH_ENV_PATH \
 && chmod 600 $SSH_ENV_PATH && source $SSH_ENV_PATH \
-&& ssh-add /etc/ssh/dwo_ssh_key < /etc/ssh/passphrase \
-&& if [ -f $HOME/.bashrc ] && [ -w $HOME/.bashrc ]; then echo "source ${SSH_ENV_PATH}" >> $HOME/.bashrc; fi; fi`
+&& if timeout 3 ssh-add /etc/ssh/dwo_ssh_key < /etc/ssh/passphrase \
```

Comment:
@vinokurig could you clarify the timeout part, looks error prone

Reply:
@ibuziuk This is need to skip the passphrase input in case if the stored passphrase is wrong. There are some elaborations in the pull request description.